packages/ui/src/elements/onClickOutside.tsx

```diff
@@ -8,4 +8,6 @@ class Comp extends React.Component<{ onClick: (e: Event) => void }> {
   }
 }
 
-export const OnClickOutside = onClickOutsideHOC(Comp)
+export const OnClickOutside: typeof Comp = (onClickOutsideHOC(
```

Comment:
why do we need to change this?

Reply:
Because the build fails otherwise. Either because of a newer TS version or `@types/react-on-click-outside`